panther_analysis_tool/main.py

```diff
@@ -758,9 +841,10 @@ def run() -> None:
     logging.basicConfig(format='[%(levelname)s]: %(message)s',
                         level=logging.DEBUG if args.debug else logging.INFO)
 
+    if hasattr(args, 'filter') and args.filter is not None:
```

Comment:
Minor, should be equivalent to `if getattr(args, 'filter', None) is not None`.

Reply:
fixed